core/src/main/java/de/danoeh/antennapod/core/service/FeedUpdateWorker.java

```diff
@@ -41,7 +40,6 @@ public Result doWork() {
             // We repeatedly schedule a OneTimeWorkRequest instead.
             AutoUpdateManager.restartUpdateAlarm(getApplicationContext());
         }
-
```

Comment:
Please do not change unrelated files - that makes the git history harder to review.